File: CSharp/Unit1/03.ConsoleInputOutput/02.CirclePerimeterArea/CirclePerimeterArea.cs
/*
 * Write a program that reads the radius r of a circle and prints its 
 * perimeter and area.
*/
using System;
class CirclePerimeterArea
{
    static void Main()
    {
        Console.Write("Input radius of the circle: ");
        double radius = double.Parse(Console.ReadLine());
        Console.WriteLine("Perimeter of circle = {0:0.00}",(2 * Math.PI * radius));
        Console.WriteLine("Area of circle = {0:0.00}",(Math.PI * radius * radius));
    }
}

File: Java/IntroJava/03.ConsoleInputOutput/src/CirclePerimeterArea.java
/*
 * Write a program that reads the radius r of a circle and prints its 
 * perimeter and area.
*/
import java.util.Scanner;
public class CirclePerimeterArea {

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		System.out.print("Input radius of the circle: ");
        double radius = input.nextDouble();
        input.close();
        System.out.printf("Perimeter of circle = %-10.2f",(2 * Math.PI * radius));
        System.out.printf("Area of circle = %10.2f",(Math.PI * radius * radius));
	}

}